net-irc/atheme-services/files/atheme-services.initd

```diff
@@ -0,0 +1,36 @@
+#!/sbin/openrc-run
+# Copyright 1999-2011 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+extra_started_commands="reload"
+
+depend() {
+	need net
+	use ircd
+	provide irc-services
+}
+
+start() {
+	ATHEME_SERVICES_RUNDIR=/var/run/atheme
+	if ! [ -d "${ATHEME_SERVICES_RUNDIR}" ]; then
+	    ebegin "Creating \`${ATHEME_SERVICES_RUNDIR}'"
+	    install -o atheme-services -d "${ATHEME_SERVICES_RUNDIR}"
```

Comment:
I think `checkpath` is the modern way of doing it. Or even better, a tmpfiles.d file that works for people not using the init.d script directly.